examples/service-clients/azure-client/external-controller/src/app.ts

```diff
@@ -71,56 +71,46 @@ const connectionConfig: AzureRemoteConnectionConfig | AzureLocalConnectionConfig
 const containerSchema = {
 	initialObjects: {
 		/* [id]: DataObject */
-		map1: SharedMap,
-		map2: SharedMap,
+		tree: SharedTree,
 	},
-} satisfies ContainerSchema;
+} as const satisfies ContainerSchema;
 type DiceRollerContainerSchema = typeof containerSchema;
 
-function createDiceRollerControllerProps(map: ISharedMap): DiceRollerControllerProps {
-	return {
-		get: (key: string) => map.get(key) as number,
-		set: (key: string, value: unknown) => map.set(key, value),
-		on(
-			event: "valueChanged",
-			listener: (args: IValueChanged) => void,
-		): DiceRollerControllerProps {
-			map.on(event, listener);
-			return this;
-		},
-		off(
-			event: "valueChanged",
-			listener: (args: IValueChanged) => void,
-		): DiceRollerControllerProps {
-			map.on(event, listener);
-			return this;
-		},
-	};
-}
+const treeViewConfig = new TreeViewConfiguration<typeof App>({
```

Comment:
Wouldn't it be better to cut the explicit type argument and stick to the default `ImplicitFieldSchema`?